docs/deployment/customization/portal.properties-Reference.md

```diff
@@ -191,13 +191,15 @@ Namespace columns are custom columns in the MAF file that can be shown in Mutati
 skin.mutation_table.namespace_column.show_by_default=
 ```
 
-### Default visible columns on init in Mutations Table
+### Default visible columns on init in Mutation, Copy-Number and Structural Variant Tables
 
-Define the columns that are going to be visible in the Mutation Table on the Patient View and the Results View. If namespace columns are set to visible by default, they will also be shown.
+Define the columns that are going to be visible in the Mutation Copy-Number and Structural Variant Tables on the Patient View and the Results View. If namespace columns are set to visible by default, they will also be shown.
```

Comment:
```suggestion
Define the columns that are going to be visible in the Mutation, Copy-Number and Structural Variant Tables on the Patient View and the Results View. The `skin.mutation_table.namespace_column.show_by_default` takes precedence over these settings for namespace columns.
```

Reply:
Also, the text states:

> the Patient View and the Results View

I do not see properties for Results View CNA and SV tables. Did you forget about these?

Reply:
@pvannierop There are no CNA and SV tables in the Results View. I have clarified this in the documentation while implementing your suggestions.